model/riscv_vmem.sail

```diff
@@ -120,6 +120,9 @@ struct SV_Params = {
   pte_PPN_j_size_bits : {10,  9}         //   10     9     9
 }
 
+// Current level during a page-table walk (0 to SV_Params.levels - 1)
+type Level = range(0,3)    // range(0,4) when add Sv57 (TODO)
```

Comment:
Level is quite a generic name. Until there is a real way of declaring this file-local in sail, I think we have to use something like PTELevel here.

Reply:
Ok, changed name to PTW_Level